### DISCUSSION TITLE
HmIP-DRG-DALI Lampensteuerung über CCU3 funktioniert, über hahomatic nicht.

### DISCUSSION THREAD

**Discussion by heini66** [SukramJ/aiohomematic#1465] [General]
Moin,
ersteinmal vielen Dank für das tolle Projekt !!!

Nu zum Problem: Ich hab das HmIP-P-DRG-DALI in der CCU3 und kann von dort die Lampen Steuern. In hahomatic sehe ich diverse Entitäten wie Sensoren (Dim2Warm State ch1-48, Human Centric Light State Ch 1-48 ), Konfiguration (  Color Temp ch n,  HUE Channel 1-48, Level ch 1-48 und Saturation ). Dimmen kann ich die Lampen aber es gibt nicht die Möglichkeit wie in der CCU3 komplett an oder auszuschalten. Kann es sein, das nicht alle Entitäten vom HmIP-P-DRG-DALI vom hahomatic ausgelesen werden?

**Comment by SukramJ** [comment]
> Kann es sein, das nicht alle Entitäten vom HmIP-P-DRG-DALI vom hahomatic ausgelesen werden?

Eine grundsätzliche Erklärung findest Du [hier](https://github.com/danielperna84/custom_homematic#device-support).
Da Du der Erste hier bist der so ein Gerät hat benötigen wir für eine Integration die Gerätebeschreibungen. Wie Du diese erstellst findest Du [hier](https://github.com/danielperna84/custom_homematic#homematicip_localexport_device_definition). Die beiden Dateien können auch hier angehängt werden.

Häng bitte noch ein paar Gerätescreenshots aus der CCU und HA hier an.

**Comment by heini66** [comment]
Danke.
Hab gerade keinen Zugriff. Morgen zur Arbytszeit wieder.
Wo finde ich eine Liste der Unterstützen Geräte im Github??

**Reply by SukramJ** [reply]
Wie du vielleicht [gelesen](https://github.com/danielperna84/custom_homematic#device-support) hast werden alle Geräte unterstützt. 
Bei manchen Geräten muss man  noch ein wenig nachhelfen, um diese als `light` oder `climate` darzustellen.

**Comment by heini66** [comment]
Moin,
benötigst Du noch mehr Bilder? 
![CCU_dali_1](https://github.com/danielperna84/hahomematic/assets/5873240/dd43bd51-0c26-4083-9053-62ae3dbee523)
![CCU_dali_2](https://github.com/danielperna84/hahomematic/assets/5873240/710e38e2-2886-4363-93e8-fd57e99f3137)
![Haos_dali](https://github.com/danielperna84/hahomematic/assets/5873240/6b3790f9-435f-47d7-bc99-09b5932483ef)
[HmIP-GRG-DALI_device_descriptions.json](https://github.com/danielperna84/hahomematic/files/14694972/HmIP-GRG-DALI_device_descriptions.json)
[HmIP-GRG-DALI-paramset_descriptions.json](https://github.com/danielperna84/hahomematic/files/14694974/HmIP-GRG-DALI-paramset_descriptions.json)

**Reply by SukramJ** [reply]
Häng bitte noch mal Bilder aus den Geräteeinstellungen dran

**Comment by heini66** [comment]
das?
![CCU_dali_einstellungen](https://github.com/danielperna84/hahomematic/assets/5873240/dbb44c67-996f-4af3-a0b0-3362017fe41d)
![CCU_dali_einstellungen_1](https://github.com/danielperna84/hahomematic/assets/5873240/0520918e-4d44-4730-8465-455519adc76c)

**Comment by Medienzentrum-Landkreis-Harburg** [comment]
@SukramJ 
bin nun endlich dazu gekommen, das ganze in Betrieb zu nehmen. Dabei ist mir aufgefallen das der Lampenkanal1 in der Homematik angezeigt wird und auch steuerbar ist, im Homeassistant fehlt leider Kanal 1 und somit ist die Lampe nicht steuerbar. Kannst Du das bitte noch anpassen?

Vielen Dank im vorraus
![chan_1_missing](https://github.com/user-attachments/assets/f28ed421-9576-4a3e-8d51-fa5409c3d287)
![chan_1_present](https://github.com/user-attachments/assets/2bf5b70b-96a2-4560-8a35-0dde1f8b9173)

**Reply by SukramJ** [reply]
> @SukramJ
bin nun endlich dazu gekommen, 

Ich mag taggen schon nicht, aber eMail finde ich absolut nervig. Lass das!
Was ist so dringend?
